server/backend/tests/db_helpers.py

```
@@ -14,19 +14,389 @@
+        
+        Returns:
+            A sequence of (date, count) pairs representing the number of reviews for each day that match the provided filters.
+        """
+        return await self.reviews.daily_counts(*args, **kwargs)
+
+    async def get_review_status(self, unit_id: str):
+        """
+        Retrieve the review status for a knowledge unit.
+        
+        Parameters:
+        	unit_id (str): ID of the knowledge unit to query.
```

Ruff (0.15.15)

[error] Indentation contains mixed spaces and tabs

(E101)